basic-2/src/test/java/Convertible_Car.java

```diff
@@ -0,0 +1,31 @@
+import java.util.jar.Attributes.Name;
+
+public class Convertible_Car extends Car {
+	public String convertibleStatus="";
+	public int ConvertibleAdjestment=0; 

```

Comment:
boolean is a better choice here for the top status since the top only has two possible states (up/down)